projects/common/src/utilities/validators/form-validators.test.ts

```diff
@@ -0,0 +1,21 @@
+import { FormControl } from '@angular/forms';
+import { CustomFormValidators } from '@hypertrace/common';
+
+// tslint:disable: no-null-keyword
+describe('CustomFormValidators', () => {
```

Comment:
nit: should we create a single test for a validator, I think this is a good use case to do that, since in the future we may have multiple validators, and keeping 1 for 1, would be easy to read. Use comments to tell what we're going to test.

"commaSeparatedEmails validator should work as expected'" or something like that??

Reply:
description contains the parent i.e., "CustomFormValidators".
The individual tests are where name of the validators like "commaSeparatedEmails" are used